cmd/ocsp-updater/main.go

```diff
@@ -26,13 +25,24 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
+/*
+ * dbMockAdapter is an interface collecting the gorp.DbMap functions that the
+ * various parts of OCSPUpdater rely on. Using this adapter shim allows tests to
+ * swap out the dbMap implementation.
+ */
+type dbMockAdapter interface {
```

Comment:
Doop, reviewing at the same time doesn't work properly, @jsha has a better name.